Open Judge System/OJS.Common/GlobalConstants.cs

```diff
@@ -118,6 +118,9 @@ public static class GlobalConstants
         public const int TestInputMinLength = 1;
         public const int TestOutputMinLength = 1;
 
+        // JVM Startup Memory
+        public const int JvmMemoryRequirement = 200000000;
```

Comment:
I believe it is good practice to include the measuring unit in the variable name (I've read that in a book), @KristianMariyanov ??